plugins/QnA/models/class.qnamodel.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Class QnaModel
+ */
+class QnaModel extends Gdn_Model {
+    /**
+     * Update Q&A counts through the dba/counts endpoint.
+     *
+     * @param string $column
+     * @return array $results Formatted to match what "dba.js" expects
+     */
+    public function counts($column) {
+        $result = ['Complete' => true];
+
+        switch($column) {
+            // Discussion table, QnA column will be updated.
+            case 'QnA':
+                $request = Gdn::request()->get();
+                $result = $this->recalculateDiscussionQnABatches($request['NumberOfBatchesDone'] ?? 0, $request['LatestID'] ?? 0);
+                break;
+        }
```

Comment:
not sure if the break is lined up properly.  In my IDE it's line up with the case, it should be lined up with $result if i'm not mistaken.

Reply:
It's lined up with $result